config.yml

```diff
@@ -1,19 +1,16 @@
 ---
+# INSTRUCTIONS
+# 'name' and 'email' are self-explanatory, but preference is used to
+# specify whether you prefer to meet synchronously or asynchronously.
+# Use the value 'sync', 'async', or 'none' to specify your preference
+#
+# For now, only add entries to 'hobbits'
 hobbits:
-  - nhooper
-  - sreddy
-  - droberts
-  - dmiller
-  - skelly
-  - esoergel
-  - biyeun
-  - cellowitz
-  - mkangia
-  - jschweers
-  - aphulera
-  - sherbers
-  - gherceg
-  
+  - name: Graham
+    email: gherceg@dimagi.com
```

Comment:
Do we want full email's here? Just thinking about email scrapers and spam

Reply:
Yeah that is a good reason not to. I'll revert that change 👍🏻

Reply:
8294f683b86664a1afc7254bc77ace9e8c402f84